components/sidebars/SidebarEditTags.js

```diff
@@ -0,0 +1,160 @@
+import * as React from "react";
+import * as Strings from "~/common/strings";
+import * as Constants from "~/common/constants";
+import * as System from "~/components/system";
+import * as Validations from "~/common/validations";
+import * as Actions from "~/common/actions";
+import * as Events from "~/common/custom-events";
+
+import { css } from "@emotion/react";
+import { Tag } from "~/components/system/components/Tag";
+
+const STYLES_GROUPING = css`
+  width: 100%;
+  border: 1px solid rgba(196, 196, 196, 0.5);
+  background-color: ${Constants.system.white};
+  border-radius: 6px;
+  padding: 16px;
+  margin-bottom: 40px;
+`;
+
+export default class SidebarEditTags extends React.Component {
+  state = {
+    tags:
+      !Array.isArray(this.props.sidebarData.commonTags) ||
+      this.props.sidebarData.commonTags?.length === 0
+        ? []
+        : this.props.sidebarData.commonTags,
+    suggestions:
+      !Array.isArray(this.props.sidebarData.suggestions) ||
```

Comment:
A cleaner way of doing this is:
```
state = {
    tags: this.props.sidebarData?.commonTags || []
}
```

The question mark after the sidebarData is just in case sidebar data is null